doc/authentication.md

```diff
@@ -4,7 +4,7 @@ relatedlinks: https://www.youtube.com/watch?v=6O0q3rSWr8A
 ---
 
 (authentication)=
-# Remote API authentication
+# API authentication
```

Comment:
The "Remote" part here was relevant because it only applies when accessing the API remotely (i.e over the network), as opposed to accessing the API over the local unix socket. What was the reason for removing it?

Reply:
Currently the side navigation lists "About authentication" and "About authorization" without any mention of either "Remote" or "API", overriding default use of the page title for the navlink texts. This PR deletes those overrides for these pages and defaults to the page titles; the intent in updating the page titles was to strike a middle ground with a text that's more descriptive than "About authentication" but more succinct than "Remote API authentication". 

Thanks for your explanation about needing to leave "Remote" in the page titles. I will revert those docs. Do you think it's as important to use "Remote" in the sidebar nav, or is "API authentication" sufficient there?

Reply:
> The "Remote" part here was relevant because it only applies when accessing the API remotely (i.e over the network), as opposed to accessing the API over the local unix socket. What was the reason for removing it?

I've not got strong opinions about that. If you would prefer for them to be consistent then it makes sense for remote to be in the sidebar nav too, but then it might not be succinct enough. This maybe why the compromise was made originally.

Maybe we could go for "Remote authentication" instead?

Reply:
Let's go ahead and make it consistent. I don't think it will be too long, and I think it would be helpful for users to see the word API there.